Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM python:3.10
+FROM python:3.10-bookworm
 LABEL description="Deploy Mage on ECS"
-ARG PIP=pip3
 USER root
 
-# Packages
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
By default, when a pipeline is executed, the exit status of the pipeline is the exit status of the last command in the pipeline. This can make it challenging to determine which command in the pipeline failed if you want to handle errors in a more granular way.

The "pipefail" option, when enabled, causes a pipeline to return the exit status of the last command to exit with a non-zero status, or to return zero if all commands in the pipeline were successful. This makes it easier to identify which command in a pipeline failed and take appropriate action based on that information.

Example
``` bash
command1 | command2 | command3
```

In the example without configuring `-o pipefail`, the exit status of the entire pipeline would be that of `command3`. If `command1` or `command2` failed, it might be challenging to identify which one caused the failure. In the second example with `-o pipefail` configured, the exit status would be that of the first command in the pipeline that failed, making it easier to determine the source of the error.

To configure pipefail is best practice.